apps/web/src/server/app/inspector/inspector.router.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
```

Comment:
The filename should be `inspector.route.js`.

Reply:
Ok, I did wonder why though as it exports a router... It would have been my third choice after *.router.js and *.routes.js !

Reply:
I don't mind either one of them. I use route out of memory muscle. If you refactor all of them in a separate PR and tell us to rebase I will 👍🏻 .

Probably .routes.js sounds better.